backend/curation/api/v1/curation/collections/common.py

```diff
@@ -288,37 +288,38 @@ def get_collection_level_processing_status(datasets: List[DatasetVersion]) -> st
     return return_status
 
 
-def get_inferred_collection_version_else_forbidden(collection_id: str) -> CollectionVersionWithDatasets:
+def get_inferred_collection_version(collection_id: str) -> CollectionVersionWithDatasets:
```

Comment:
the name of this method should retain `_else_forbidden` suffix since it still throws `ForbiddenHTTPException`

Reply:
It can also throw NotFoundHTTPException though

Reply:
unless we want to throw Forbidden for all use cases? But we've been using NotFound in other endpoints

Reply:
ah fair point. Ok, fine to remove if we aren't being consistent.